client/components/Users/UserInfoField.jsx

```diff
@@ -3,8 +3,7 @@ import './UserInfoField.css';
 
 export default class UserInfoField extends Component {
   static propTypes = {
-    title: PropTypes.string.isRequired,
-    children: PropTypes.object.isRequired
+    title: PropTypes.string.isRequired
```

Comment:
I removed it because in console we have a lot of errors

 Warning: Failed propType: Required prop `children` was not specified in `UserInfoField`. Check the render method of `UserInfo`.

arning: Failed propType: Invalid prop `children` of type `string` supplied to `UserInfoField`, expected `object`. Check the render method of `UserInfo`.

children could be string or number and I guess there is no need to prop.validate this attribute